mysql/mysqld.cnf

```diff
@@ -4,4 +4,7 @@ general_log=1
 # Slow query settings:
 slow_query_log=1
 slow_query_log_file=/var/log/mysql/slow.log
-long_query_time=5.0
\ No newline at end of file
+long_query_time=5.0
+
+# Error logging
+log_error=/var/lib/mysql/mysql_error.log
```

Comment:
It would be better to log this in the `/var/log/mysql` directory for consistency.

Reply:
But right now all logs are in `/var/lib/mysql` folder

Reply:
What other logs exist? Can we move them all to `/var/log` like the slow log?
![image](https://github.com/metakgp/metakgp-wiki/assets/34770591/ff290b79-5e63-435b-9509-e8449eced035)

Reply:
I think there is some issue with write permission in `log/` directory. The official page also uses `lib/` folder. Maybe we should move the `slow_log` to `lib/`

Reply:
Okay.

Reply:
Done!